galgebra/mv.py

```diff
@@ -1668,10 +1668,7 @@ def is_scalar(self):
 
     def components(self):
         return tuple(
-            Dop(dop._consolidate_terms(
-                (Mv(coef * base, ga=self.Ga), pdiff)
-                for (coef, pdiff) in sdop.terms
-            ), ga=self.Ga)
+            Dop([(sdop, Mv(base, ga=self.Ga))], ga=self.Ga)
```

Comment:
Lines that this duplicated are here:

https://github.com/pygae/galgebra/blob/07d8ebc7a4b39c6940aa94d1fe96f9d7e3b0f302/galgebra/mv.py#L1467-L1471